plugins/connection/pct_remote.py

```diff
@@ -0,0 +1,396 @@
+        cli:
+          - name: ssh_extra_args
+        default: ''
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      pty:
+        default: True
+        description: 'SUDO usually requires a PTY, True to give a PTY and False to not give a PTY.'
+        env:
+          - name: ANSIBLE_PARAMIKO_PTY
+        ini:
+          - section: paramiko_connection
+            key: pty
+        type: boolean
+      record_host_keys:
+        default: True
+        description: 'Save the host keys to a file'
+        env: [{name: ANSIBLE_PARAMIKO_RECORD_HOST_KEYS}]
+        ini:
+          - section: paramiko_connection
+            key: record_host_keys
+        type: boolean
+      host_key_checking:
+        description: 'Set this to "False" if you want to avoid host key checking by the underlying tools Ansible uses to connect to the host'
+        type: boolean
+        default: True
+        env:
+          - name: ANSIBLE_HOST_KEY_CHECKING
+          - name: ANSIBLE_SSH_HOST_KEY_CHECKING
+          - name: ANSIBLE_PARAMIKO_HOST_KEY_CHECKING
+        ini:
+          - section: defaults
+            key: host_key_checking
+          - section: paramiko_connection
+            key: host_key_checking
+        vars:
+          - name: ansible_host_key_checking
+          - name: ansible_ssh_host_key_checking
+          - name: ansible_paramiko_host_key_checking
+      use_persistent_connections:
+        description: 'Toggles the use of persistence for connections'
+        type: boolean
+        default: False
+        env:
+          - name: ANSIBLE_USE_PERSISTENT_CONNECTIONS
+        ini:
+          - section: defaults
+            key: use_persistent_connections
+      banner_timeout:
+        type: float
+        default: 30
+        description:
+          - Configures, in seconds, the amount of time to wait for the SSH
+            banner to be presented. This option is supported by paramiko
+            version 1.15.0 or newer.
+        ini:
+          - section: paramiko_connection
+            key: banner_timeout
+        env:
+          - name: ANSIBLE_PARAMIKO_BANNER_TIMEOUT
+      timeout:
+        type: int
+        default: 10
+        description: Number of seconds until the plugin gives up on failing to establish a TCP connection.
+        ini:
+          - section: defaults
+            key: timeout
+          - section: ssh_connection
+            key: timeout
+          - section: paramiko_connection
+            key: timeout
+        env:
+          - name: ANSIBLE_TIMEOUT
+          - name: ANSIBLE_SSH_TIMEOUT
+          - name: ANSIBLE_PARAMIKO_TIMEOUT
+        vars:
+          - name: ansible_ssh_timeout
+          - name: ansible_paramiko_timeout
+        cli:
+          - name: timeout
+      private_key_file:
+          description:
+              - Path to private key file to use for authentication.
+          type: string
```

Comment:
I believe this should be a:
```suggestion
          type: path
```

Reply:
I copied this from `ssh_paramiko`: https://github.com/ansible/ansible/blob/347a822a42c7a2f31394e112fd6645d76d939b14/lib/ansible/plugins/connection/paramiko_ssh.py#L423C40-L423C50

It is handed over to `os.path.expanduser()` which should also be fine with `path`